clients/packages/canary-client/src/scenes/Community/Integrations/ForceSync.tsx

```diff
@@ -28,22 +43,49 @@ const ForceSync: React.FC = () => {
     { variables: { is_community: true, id: community?.id }, skip: true }
   );
 
+  const { data, loading, error }  = useQuery(
+    fetchGraphqlQuery,
+    {
+      variables: { is_community: true, id: community?.id },
+      pollInterval: 5000,
+    },
+  )
+
   const done = async () => {
     const a: MailchimpStart = await setPropagating();
     if (typeof a !== 'undefined' && 
         typeof a.data.resync_mailchimp_start.status !== 'undefined' &&
         a.data.resync_mailchimp_start.status === 'started to add contacts to the queue') {
-      toast(<Success message={`Ae! Sincronização em andamento.`} />, { type: toast.TYPE.SUCCESS });
+      toast(<Success message={`Ae! Sincronização em andamento.`} />, { type: toast.TYPE.SUCCESS }); 
     } else {
       toast(`Falha na atualização da base de contatos do mailchimp! ${a.data.resync_mailchimp_start.status}`, { type: toast.TYPE.ERROR });
     }
   }; 
 
-  
+  if (loading) return <Text>Carregando Mailchimp Status</Text>;
+  else if (error) return <Text>Failed: {error.message}</Text>;
+
+  if (data.resync_mailchimp_status.waiting > 0){
+    return <Stack>
+      <Heading as="h4" size="sm">Forçar sincronização</Heading>
+      <Text>Sua base no Mailchimp não está atualizada? Tudo bem! Clique em sincronizar pra dar um empurrãozinho:</Text>
+      <Heading as="h4" size="sm">Status</Heading>
+      <Text size="sm">{data.resync_mailchimp_status.status} ({data.resync_mailchimp_status.completed})</Text>
+      <Flex justifyContent="flex-end">
+        <Button onClick={done} disabled='true' type='button' marginTop={4}>Sincronizar</Button>
+      </Flex>
+    </Stack>
+  } 
+
   return (
     <Stack>
       <Heading as="h4" size="sm">Forçar sincronização</Heading>
       <Text>Sua base no Mailchimp não está atualizada? Tudo bem! Clique em sincronizar pra dar um empurrãozinho:</Text>
+      <Heading as="h4" size="sm">Status</Heading>
+      <Text size="sm">{data.resync_mailchimp_status.status}</Text>
+      <Text size="sm">{((typeof data.resync_mailchimp_status.last_sync === 'undefined' || data.resync_mailchimp_status.last_sync === "")? '' 
+                      : `Data da última atualização: ${data.resync_mailchimp_status.last_sync}`)}
```

Comment:
é preferível, evitar essas verificações dentro da renderização ou seja, no retorno dos elementos que vão renderizar na tela para simplificar a leitura do código.

Se não tiver mais nenhuma alteração, podemos subir assim mas se for subir algo mais, vale mover essa verificação para uma função e só chamar aqui.

Reply:
Acho que entendi, tem mais uns detalhes para acertar vou tentar fazer isso também